### PR TITLE
115 typescript cypress

### LINKED ISSUE #102: Update query client config
Following this change https://github.com/graasp/graasp-query-client/pull/174 the query client config should be updated

### LINKED ISSUE #115: Add typescript

### PR DESCRIPTION
**Pull request by pyphilia**
This PR adds typescript and cypress (only one mock test).

close #115 
close #102

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```
Comment (dialexo) [inline cypress/support/commands.ts:1]: Is this file commented out for future reference when setting up our own commands?

File: package.json [hunk 2]
```diff
@@ -41,21 +41,26 @@
...
     "prettier:check": "prettier --check 'src/**/*.js'",
     "prettier:write": "prettier --write 'src/**/*.js'",
-    "release": "git fetch --tags && git add CHANGELOG.md && standard-version -a",
-    "release:first": "git fetch --tags && git add CHANGELOG.md && standard-version -a --first-release",
-    "postinstall": "husky install"
+    "release": "standard-version -a",
+    "release:first": "standard-version -a --first-release",
+    "release:minor": "standard-version -a --release-as minor",
+    "release:major": "standard-version -a --release-as major",
+    "release:patch": "standard-version -a --release-as patch",
```
Comment (dialexo) [inline package.json:60]: Are we replacing those by release-please? @spaenleh

**Review by dialexo — APPROVED** [review]
LGTM 🫕 Just left a few questions

File: package.json (hunk 2 above)
Comment (spaenleh) [reply package.json:60]: @dialexo Yes, i think this will be the way to go, but not ready yet, it will require a configurable workflow in `.github` repo.

**Pull request merged**